handler/secret/secret_test.go

```diff
@@ -162,29 +161,19 @@ func TestDecrypt(t *testing.T) {
 	})
 }
 
-func TestWrite(t *testing.T) {
+func TestWriteFails(t *testing.T) {
 	withEnv(func() {
 		Convey("write functions as expected", t, func() {
```

Comment:
this test case is probably not `write functions as expected`? probably more of a `given a failure writing to vault`